beep/tests/test_structure.py

```diff
@@ -343,6 +361,29 @@ def test_get_interpolated_charge_step(self):
         self.assertTrue(np.all(np.array(lengths) == 1000))
         self.assertTrue(interpolated_charge["current"].mean() > 0)
 
+    def test_whether_maccor_step_is_waveform(self):
```

Comment:
Could we make this test case a little more complete? Basically, if there were arbin files that failed since voltage was not monotonic it would be good to have them here. I would really like to move away from using the `_wf` columns

Reply:
Even just including the Arbin files that didn't pass the monotonicity test in a comment or something would be helpful